src/pages/edit-test-center.page.js

```diff
@@ -0,0 +1,474 @@
+import React, { Component } from 'react';
+import { bindAll, cloneDeep } from 'lodash';
+import styled from 'styled-components'; 
+
+import FacilityService from '@services/facility.service.js';
+
+import {
+  Container,
+  TextField,
+  MenuItem,
+  Select,
+  RadioGroup,
+  Radio,
+  FormControlLabel,
+  FormControl,
+} from '@material-ui/core';
+
+import { AppContext } from '@contexts/app.context';
+import {
+  GOT_TESTED_OPTIONS,
+  OFFERINGS,
+  SCREENING_METHODS,
+  POST_DATA_STATE,
+} from '@constants/add-test-center.constants';
+import Header from '@general/headers/header';
+import PrimaryButton from '@general/buttons/primary-button';
+import SnackbarMessage from '@general/alerts/snackbar-message';
+import TypesService from '@services/types.service.js';
+import FacilitateService from '@services/facilitate.service.js';
+
+export default class EditTestCenterPage extends Component {
```

Comment:
this is basically a copy-paste job from addtestcenterpage. you should extend that component, and just change the way the data is populated instead of just making a copy like this

Reply:
Yes that's a good idea. It seems the only thing that change are the initial state and the way it is being submitted. I can put this into a different component to avoid any duplicate code issues.

Reply:
I created a component (test-center-update.js) which takes in the data to be populated from add-test-center and edit-test-center. Functions that are handled differently or might be in the future are also passed in to the component.